### PR TITLE
update links

### PR DESCRIPTION
**Pull request by Jaime-Trinidad**
#### Summary

Update links for device repository page, attends #302 

#### Changes

- ...
- ...

#### Notes for Reviewers

...

#### Release Notes

- ...

### CODE REVIEW

**Comment by Jaime-Trinidad** [comment]
@iulia-branescu can you approve this updates, I change for the right links

File: vendor/tektelic/t00048xx-smart-room-pir.yaml [hunk 1]
```diff
@@ -64,8 +64,8 @@ keyProvisioning:
 keySecurity: none
 
 # Product and data sheet URLs (optional)
-productURL: https://tektelic.com/public/catalog/Smart-Room-PIR-Sensor
```
Comment (iulia-branescu) [inline vendor/tektelic/t00048xx-smart-room-pir.yaml:67]: this doesn't work

Comment (Jaime-Trinidad) [reply vendor/tektelic/t00048xx-smart-room-pir.yaml:67]: yes, that is the one I change

File: vendor/tektelic/t00048xx-smart-room-pir.yaml [hunk 2]
```diff
@@ -64,8 +64,8 @@ keyProvisioning:
...
-dataSheetURL: https://tektelic.com/public/uploads/Brochures/Smart%20Room%20Sensor.pdf
+productURL: https://tektelic.com/catalog/smart-room-lorawan-sensor-pir
```
Comment (Jaime-Trinidad) [inline vendor/tektelic/t00048xx-smart-room-pir.yaml:67]: this is the new one

Comment (iulia-branescu) [reply vendor/tektelic/t00048xx-smart-room-pir.yaml:67]: ok nice, thank you!

**Review by iulia-branescu — APPROVED** [review]

**Comment by Jaime-Trinidad** [comment]
Hi @nejraselimovic can you approve this PR, are some links updates. Thanks

**Review by nejraselimovic — APPROVED** [review]

**Pull request merged**